openhands/events/action/message.py

```diff
@@ -11,6 +11,7 @@ class MessageAction(Action):
     wait_for_response: bool = False
     action: str = ActionType.MESSAGE
     security_risk: ActionSecurityRisk | None = None
+    secondary_id: str | None = None
```

Comment:
Sorry, I still don't get why this needs a `secondary_id`. It's an event, so it has an event_id once it gets in the stream. Doesn't it need to get in the stream anyway?

Reply:
This is now outdated - we no longer have a `secondary_id` but instead have a pending flag for outgoing messages that exists only on the client. Once a message is echoed from the server, all pending messages are cleared.

Reply:
That sounds good to me, so events are identified by a single `id` - some sooner, some after the user sees them, but soon enough as long as connected.